src/app/app.html

```diff
@@ -22,17 +22,17 @@
             </label>
         </form>
         <div class="header-actions">
-            <clr-dropdown [clrMenuPosition]="'bottom-right'">
-                <button class="nav-icon" clrDropdownToggle>
-                    <clr-icon shape="cog"></clr-icon>
-                    <clr-icon shape="caret down"></clr-icon>
-                </button>
-                <clr-dropdown-menu>
-                    <a href="javascript://" clrDropdownItem>About</a>
-                    <a href="javascript://" clrDropdownItem>Preferences</a>
-                    <a href="javascript://" clrDropdownItem>Log out</a>
-                </clr-dropdown-menu>
-            </clr-dropdown>
+            <!--<clr-dropdown [clrMenuPosition]="'bottom-right'">-->
```

Comment:
My bad, this is clean-up that I missed on my second commit. Thanks for pointing it out, I'm rolling back to the current state in master.